lib/order.rb

```diff
@@ -0,0 +1,100 @@
+require "csv"
+require_relative "customer"
+
+class Order
+  attr_reader :id, :products, :customer, :fulfillment_status, :total
+  
+  def initialize(id, products, customer, fulfillment_status=:pending)
+    @id = id.to_i
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status 
+    
+    valid_statuses = [:pending, :paid, :processing, :shipped, :complete]
+    
+    if !valid_statuses.include?(fulfillment_status)
+      raise ArgumentError.new("That is not a valid fulfillment status.")
+    end
+  end
+  
+  def total
+    subtotal = products.sum do |product, price|
+      price
+    end
+    
+    total = subtotal * 1.075
```

Comment:
I think this might not be doing what you think it does, based on your code above.